src/centos/scripts/start-marklogic.sh

```diff
@@ -322,10 +334,21 @@ else
 fi
 
 ################################################################
-# mark the node ready
+# check if node is available and mark it ready
 ################################################################
-info "Cluster config complete, marking this node as ready."
-sudo touch /var/opt/MarkLogic/ready
+HOST_RESP_CODE=""
```

Comment:
HOST_RESP_CODE="" is not necessary here.

Reply:
var HOST_RESP_CODE earlier was used for validating the host and as a best practice, initialized it with an empty string. It can be removed as it's not used anywhere else in the script right now.